start-picsure.sh

```diff
@@ -7,16 +7,18 @@
 # how that does or doesn't change the commands you run when inside Jenkins
 
 DOCKER_CONFIG_DIR="${DOCKER_CONFIG_DIR:-/usr/local/docker-config}"
+# Use this for file system checks. Use DOCKER_CONFIG_DIR for docker commands.
+# Except for --env_file commands, which refer to the current file system, not the root fs
+CURRENT_FS_DOCKER_CONFIG_DIR="${CURRENT_FS_DOCKER_CONFIG_DIR:$-DOCKER_CONFIG_DIR}"
 
-if [ -f "$DOCKER_CONFIG_DIR/setProxy.sh" ]; then
-   . $DOCKER_CONFIG_DIR/setProxy.sh
+if [ -f "$CURRENT_FS_DOCKER_CONFIG_DIR/setProxy.sh" ]; then
+   . $CURRENT_FS_DOCKER_CONFIG_DIR/setProxy.sh
 fi
 
-if [ -z "$(grep "VITE_ALLOW_EXPORT" $DOCKER_CONFIG_DIR/httpd/.env | grep 'false')" ]; then
-    export EXPORT_SIZE="2000";
-  else
-    export EXPORT_SIZE="0";
-  fi
+if [ -z "$(grep "VITE_ALLOW_EXPORT" $CURRENT_FS_DOCKER_CONFIG_DIR/httpd/.env | grep 'false')" ]; then
+  export EXPORT_SIZE="2000";
+else
+  export EXPORT_SIZE="0";
 fi
```

Comment:
This is the snipped we talked about last week that is not actually doing anything, and the logic also seems backwards

Reply:
I think you are right about it being backwards